1559646001131	Sync.LogManager	DEBUG	Flushing file log
1559646001140	Sync.LogManager	DEBUG	Log cleanup threshold time: 1558782001140
1559646001194	Sync.LogManager	DEBUG	Done deleting files.
1559647867168	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559647867298	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559647867298	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559647867299	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559647867299	Sync.Service	INFO	Starting sync at 2019-06-04 13:31:07 in browser session DLDB2kz0MYXZ
1559647867299	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559647867300	Sync.Status	INFO	Resetting Status.
1559647867300	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1559647867303	Sync.BrowserIDManager	INFO	Getting keys
1559647867303	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1559647867303	FirefoxAccounts	DEBUG	enter getAssertion()
1559647867303	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1559647867306	Services.Common.RESTRequest	TRACE	local time, offset: 1559647867306, -355
1559647867307	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1559647867307	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559647867307	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1559647867307	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: de,en-us,en
1559647867307	Services.Common.RESTRequest	DEBUG	POST Length: 888
1559647867307	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"9A0738E04B52F9BF6655458E43DD636E2F1B5EBC15CA43F50C05590A6C3D1165CB3C2AC27521A92AFF2E72A96A77A760BBAA9CC9B02772325ED42E09DDB574E9476CA23CEEF608338AF42228765F0E7FFFDF6268E0B01BEBB9B0BC0FB9476A93C70BC6121F645934ABC0021D776EC9FAEC9117BDE5C74A107FA7716698D80732","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1559647867761	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559647868204	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559647868204	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559647868205	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559647868686	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559647868823	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559647868823	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559647868824	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559647868824	Sync.Service	INFO	Starting sync at 2019-06-04 13:31:08 in browser session DLDB2kz0MYXZ
1559647868842	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559647868843	Sync.Service	INFO	Cannot start sync: already syncing?
1559647871046	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559649793097	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559649793252	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559649793252	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559649793253	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559649793253	Sync.Service	INFO	Starting sync at 2019-06-04 14:03:13 in browser session DLDB2kz0MYXZ
1559649793253	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559649793253	Sync.Service	INFO	Cannot start sync: already syncing?
1559649793761	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559649793939	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559649793939	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559649793940	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559649794694	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559649795831	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559649795831	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559649795834	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559649795834	Sync.Service	INFO	Starting sync at 2019-06-04 14:03:15 in browser session DLDB2kz0MYXZ
1559649795840	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559649795840	Sync.Service	INFO	Cannot start sync: already syncing?
1559649795872	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1559649795872	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398862
1559649795872	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398862 - NS_ERROR_NET_TIMEOUT
1559649795872	Services.Common.RESTRequest	TRACE	POST body: 
1559649795872	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_NET_TIMEOUT"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:434
1559649795872	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
request()@resource://services-common/hawkclient.js:224
1559649795872	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
request()@resource://services-common/hawkclient.js:227
1559649795872	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1559649795872	Services.Common.RESTResponse	TRACE	Processing response headers.
1559649795872	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:234
1559649795873	Services.Common.RESTResponse	TRACE	Processing response headers.
1559649795873	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:235
1559649795873	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
_constructError()@resource://services-common/hawkclient.js:116
request()@resource://services-common/hawkclient.js:240
1559649795873	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:117
request()@resource://services-common/hawkclient.js:240
1559649795873	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:118
request()@resource://services-common/hawkclient.js:240
1559649795873	Services.Common.RESTResponse	TRACE	Processing response headers.
1559649795873	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:240
1559649795873	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_NET_TIMEOUT\"  nsresult: \"0x804b000e (NS_ERROR_NET_TIMEOUT)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559649795873	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_NET_TIMEOUT\"  nsresult: \"0x804b000e (NS_ERROR_NET_TIMEOUT)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559649795873	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_NET_TIMEOUT\"  nsresult: \"0x804b000e (NS_ERROR_NET_TIMEOUT)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559649795873	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1559649795873	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1559649795873	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: {"error":{},"errorString":"[Exception... \"NS_ERROR_NET_TIMEOUT\"  nsresult: \"0x804b000e (NS_ERROR_NET_TIMEOUT)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559649795874	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1559649795874	FirefoxAccounts	DEBUG	getOAuthToken enter
1559649795878	FirefoxAccounts	TRACE	getCachedToken returning cached token
1559649795878	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1559649795878	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1559649795878	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559649795878	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1559649795878	Services.Common.RESTRequest	TRACE	HTTP Header if-none-match: "1d3cb6698a7af7b246a9837b970c1e724afce599-gzip"
1559649796157	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559649796567	Sync.Resource	DEBUG	GET fail 401 https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections
1559649796567	Sync.Resource	WARN	GET request to https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections failed with status 401
1559649796567	Sync.Service	INFO	Logging out
1559649796568	Sync.ErrorHandler	INFO	Got 401 response; resetting clusterURL.
1559649796568	Sync.ErrorHandler	DEBUG	New mid-sync 401 failure. Making a note.
1559649796568	Sync.ErrorHandler	INFO	Attempting to schedule another sync.
1559649796569	Sync.Service	ERROR	Aborting sync: failed to get collections.
1559649796569	Sync.ErrorHandler	ERROR	Sync encountered an error: {"data":"0","url":"https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections","status":401,"success":false,"headers":{"connection":"keep-alive","content-length":"1","content-type":"application/json; charset=UTF-8","date":"Tue, 04 Jun 2019 12:03:17 GMT","server":"openresty/1.9.7.3","strict-transport-security":"max-age=15768000; includeSubDomains","www-authenticate":"Hawk","x-weave-timestamp":"1559649797.22"},"obj":0}
1559649796570	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1559649796573	Sync.Service	DEBUG	Exception calling anonymous function: {"data":"0","url":"https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections","status":401,"success":false,"headers":{"connection":"keep-alive","content-length":"1","content-type":"application/json; charset=UTF-8","date":"Tue, 04 Jun 2019 12:03:17 GMT","server":"openresty/1.9.7.3","strict-transport-security":"max-age=15768000; includeSubDomains","www-authenticate":"Hawk","x-weave-timestamp":"1559649797.22"},"obj":0}
